AirCasting/MobilePeripheralSessionManager.swift

```diff
@@ -17,7 +17,13 @@ class MobilePeripheralSessionManager {
     }
     
     func startRecording(session: Session, peripheral: CBPeripheral) throws {
-        try measurementStreamStorage.createSession(session)
+        measurementStreamStorage.accessStorage { storage in
+            do {
+                try storage.createSession(session)
+            } catch {
+                Log.info("\(error)")
+            }
+        }
         locationProvider.requestLocation()
         activeMobileSession = MobileSession(peripheral: peripheral, session: session)
```

Comment:
in case of an error in creating session - will this be a problem (nil session passed in here)?

Reply:
Hmm, not sure. Maybe I can wrap this in if let, like this: 

```
        if let session = Session(uuid: sessionUUID,
                                 type: sessionType,
                                 name: sessionContext.sessionName,
                                 deviceType: sessionContext.deviceType,
                                 location: startingLocation,
                                 startTime: Date(),
                                 contribute: contribute) {
            do {
                try microphoneManager.startRecording(session: session)
                completion(.success(()))
            } catch {
                assertionFailure("Can't start recording microphone session: \(error)")
                completion(.failure(error))
            }
        }
```

Reply:
True, what would you say about sth like this: 
```
        measurementStreamStorage.accessStorage { [weak self] storage in
            do {
                try storage.createSession(session)
                DispatchQueue.main.async {
                    self?.locationProvider.requestLocation()
                    self?.activeMobileSession = MobileSession(peripheral: peripheral, session: session)
                }
            } catch {
                Log.info("\(error)")
            }
        }
```